### ISSUE TITLE
 Microsoft surface book2，mouse click is invalid

### ISSUE THREAD

**Issue by EagleWolf** [kamranahmedse/driver.js#191]
Microsoft surface book2，mouse click is invalid。

There is no problem with touch click。

360 Secured Browser

```

/**
   * Binds any DOM events listeners
   * @todo: add throttling in all the listeners
   * @private
   */
  bind() {
    this.window.addEventListener('resize', this.onResize, false);
    this.window.addEventListener('keyup', this.onKeyUp, false);
 
    // Binding both touch and click results in popup getting shown and then immediately get hidden.
    // Adding the check to not bind the click event if the touch is supported i.e. on mobile devices
    // Issue: https://github.com/kamranahmedse/driver.js/issues/150
    if (!('ontouchstart' in document.documentElement)) {
      this.window.addEventListener('click', this.onClick, false);
    }
 
    this.window.addEventListener('touchstart', this.onClick, false);
  }

```

Can you support both mouse and touch events？

**Comment by EagleWolf** [comment]
There is no problem with touch and click

**Comment by kamranahmedse** [comment]
This should be fixed in versions after `0.9.6`. Please re-open if the issue persists.
